nistats/first_level_model.py

```diff
@@ -311,17 +312,17 @@ def fit(self, run_imgs, events=None, confounds=None,
             expected per run_img. Ignored in case designs is not None.
             If string, then a path to a csv file is expected.
 
-        confounds: pandas Dataframe or string or list of pandas DataFrames or
+        regressors: pandas Dataframe or string or list of pandas DataFrames or
                    strings,
-            Each column in a DataFrame corresponds to a confound variable
+            Each column in a DataFrame corresponds to a regressor variable
             to be included in the regression model of the respective run_img.
             The number of rows must match the number of volumes in the
             respective run_img. Ignored in case designs is not None.
             If string, then a path to a csv file is expected.
 
         design_matrices: pandas DataFrame or list of pandas DataFrames,
             Design matrices that will be used to fit the GLM. If given it
-            takes precedence over events and confounds.
+            takes precedence over events and regressors.
```

Comment:
> If you would like to contribute it, this should come as a specific PR.

Ok, may be I can give that a try after some time.

>Note however that in spite of the 'confounds' term, these regressors are actually handled as well as the others in the glm and that you can make inference upon them if you wish or need.

Yes, that was the sens of my question. I found the word `confounds` a little misleading for it may suggest the opposite of what you are pointing to (actually it frightened me w.r.t. some old analyses, although I was almost sure all the regressors are treated the same way). To be honest I personally preferred the word `regressors`, or at least `covariates`. On the other hand I agree with modality specific models strategy.